exporter/importables.go

```diff
@@ -148,7 +148,7 @@ var resourcesMap map[string]importable = map[string]importable{
 			}
 			// libraries installed with init scripts won't be exported.
 			b := body.AppendNewBlock("resource", []string{r.Resource, r.Name}).Body()
-			relativeFile := fmt.Sprintf("${path.module}/files/%s", fileName)
+			relativeFile := fmt.Sprintf("${path.module}/%s", fileName)
```

Comment:
won't it create a mix of too many files in the output location?

Reply:
There'll be two folders - files and notebooks.